scaleway/helpers.go

```diff
@@ -151,6 +151,17 @@ func parseRegionalID(regionalID string) (region scw.Region, id string, err error
 	return
 }
 
+// parseRegionalNestedID parses a regionalNestedID and extracts the resource region, inner and outer ID.
+func parseRegionalNestedID(regionalNestedID string) (region scw.Region, outerID, innerID string, err error) {
+	locality, innerID, outerID, err := parseLocalizedNestedID(regionalNestedID)
+	if err != nil {
+		return
+	}
+
+	region, err = scw.ParseRegion(locality)
+	return
```

Comment:
Could you be explicit in the variable you return? It makes IDE refactoring easier.

Reply:
This is a copy of `parseZonedNestedID`